weblate_web/templates/about.html

```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
                 </div>
             </div>
 
+            <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
+            <div class="weblate-team">
+                <div class="weblate-team-content">
+                   <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                     <div class="content-p">
+                       <p>MICHAL CIHAR</p>
```

Comment:
```suggestion
                       <p>Michal Čihař</p>
```

Please add CSS for uppercasing the stirng.

Reply:
Upercasing css has been added in both style.css and style-rtl.css